README.md

```diff
@@ -12,6 +12,15 @@ $ npm install
 $ npm run docs
 ```
 
+### Previewing
+
+We use [Yalc](https://github.com/whitecolor/yalc) to preview changes made to Molekule in consuming applications.
+
+1. Install Yalc: `yarn global add yalc`.
+1. Run `yarn preview` (which runs `yalc push && nodemon -x 'yalc push --changed'`) to publish the changes to your local `~/.yalc` directory, and re-publish when changes are detected.
+1. Run `yalc link molekule` in the consuming application, and start (or restart) that application to preview the changes locally.
```

Comment:
I ran these instructions locally to fix a bug in Molekule and the link command did not update my `package.json`.  I ended up having to use `yalc add molekule` to get it to update correctly.

Reply:
Thanks, updated.

Reply:
I misread your comment earlier @schlegz , `yalc link` shouldn't update `package.json`. Were you unable to test changes this way? (Or is the preference to update `package.json`.)

Reply:
When I tried `yalc link` it didn't seem to be pulling from my copy of `molekule` which is why I used `yalc add`.

Reply:
Thanks @schlegz. I'll merge for now, since this works for me, we can revisit if this issue resurfaces.

Reply:
@schlegz the watch script has been updated to specify a watch directory, and yalc push has been reverted back to yalc publish --push, so that the prepublishOnly script still gets run (push doesn't run other script by default).

Thanks for testing this out, and identifying issues. These should be resolved now, PTAL when convenient.